libraries/service/src/main/java/org/holochain/androidserviceruntime/service/HolochainService.kt

```diff
@@ -595,7 +634,7 @@ class HolochainService : Service() {
         Log.d(logTag, "stopForeground")
 
         // Shutdown conductor
-        runBlocking { runtime?.stop() }
+        runBlocking { runtime!!.stop() }
```

Comment:
What's the reasoning behind this change? To catch, at development time, cases where `runtime` would be null? If `runtime` would actually indeed be null at runtime (if possible at all) then might throwing an error here prevent from ever stopping the foreground?

Reply:
Yeah no good reason. I should be handling the null case properly with an error. I'll do that across the codebase in a follow up PR.